sql/files/defaultdata/kt/run

```diff
@@ -19,7 +19,7 @@ COMPILESCRIPTDIR="$(dirname "$0")"
 # Note that you then also might want to fix this in the compiler and runner version commands.
 # For example
 # KOTLIN_DIR=/usr/lib/kotlinc/bin
-KOTLIN_DIR="$(dirname "$(command -v kotlinc)")"
+KOTLIN_DIR="$(if [ -L "$(command -v kotlinc)" ]; then echo "$(dirname "$(readlink "$(command -v kotlinc)")")"; else echo "$(dirname "$(command -v kotlinc)")"; fi)"
```

Comment:
I like the idea but maybe we should split this on multiple lines?
```suggestion
if [ -L "$(command -v kotlinc)" ]
then
    KOTLIN_DIR=$(dirname "$(readlink "$(command -v kotlinc)")")
else
    KOTLIN_DIR=$(dirname "$(command -v kotlinc)")
fi
```

Seems a bit more readable to me?

Reply:
OK nvm your fix is even better.

Reply:
Checkout the new commit; I tried to keep it on a single line for easier replacement by the user (as stated in the example comments above).